docs/diagrams/sequenceDiagrams/AddQuantityToItemCommandSequenceDiagram.puml

```diff
@@ -0,0 +1,80 @@
+@startuml
+!include ../style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":InventoryinatorParser" as InventoryinatorParser LOGIC_COLOR
+participant ":AddQuantityToItemCommandParser" as AddQuantityToItemCommandParser LOGIC_COLOR
+participant ":AddQuantityToItemCommand" as AddQuantityToItemCommand LOGIC_COLOR
+participant ":EditItemCommand" as EditItemCommand LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+
+[-> LogicManager : execute("addq -n banana -q 50")
+activate LogicManager
+
+LogicManager -> InventoryinatorParser : parseCommand("addq -n banana -q 50")
+activate InventoryinatorParser
+
+create AddQuantityToItemCommandParser
+InventoryinatorParser -> AddQuantityToItemCommandParser
+activate AddQuantityToItemCommandParser
+
+AddQuantityToItemCommandParser --> InventoryinatorParser
+deactivate AddQuantityToItemCommandParser
+
+InventoryinatorParser -> AddQuantityToItemCommandParser : parse("addq -n banana -q 50")
+activate AddQuantityToItemCommandParser
+
+create AddQuantityToItemCommand
+AddQuantityToItemCommandParser -> AddQuantityToItemCommand
+activate AddQuantityToItemCommand
+
+AddQuantityToItemCommand --> AddQuantityToItemCommandParser : d
+deactivate AddQuantityToItemCommand
+
+AddQuantityToItemCommandParser --> InventoryinatorParser : d
+deactivate AddQuantityToItemCommandParser
+'Hidden arrow to position the destroy marker below the end of the activation bar.
+AddQuantityToItemCommandParser -[hidden]-> InventoryinatorParser
+destroy AddQuantityToItemCommandParser
+
+InventoryinatorParser --> LogicManager : d
+deactivate InventoryinatorParser
+
+LogicManager -> AddQuantityToItemCommand : execute()
+activate AddQuantityToItemCommand
+
+create EditItemCommand
+AddQuantityToItemCommand -> EditItemCommand : EditItemDescriptor
+activate EditItemCommand
+
+EditItemCommand -> EditItemCommand : createEditedItem(itemBanana, editItemDescriptor)
+
+EditItemCommand -> Model : setItem(itemBanana, editedItemBanana)
+activate Model
+
+Model --> EditItemCommand
+deactivate Model
+
+create CommandResult
+EditItemCommand -> CommandResult
+activate CommandResult
+
+CommandResult --> EditItemCommand
+deactivate CommandResult
+
+EditItemCommand --> AddQuantityToItemCommand
+deactivate EditItemCommand
+
+AddQuantityToItemCommand --> LogicManager : result
+deactivate AddQuantityToItemCommand
+
+destroy CommandResult
+[<--LogicManager
```

Comment:
Command Result is Returned and parsed to the UI i think So its not destroyed

<img width="854" alt="Screenshot 2020-10-19 at 9 00 17 AM" src="https://user-images.githubusercontent.com/35429757/96390978-a5b3ac80-11e9-11eb-9fea-a92b866ef24f.png">